runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -250,10 +250,11 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 
 			chatboxInputManager.openInputWindow(itemName + " tags:", initialValue, (newTags) ->
 			{
-				if (newTags == null)
+				if (newTags == null || tabInterface.isHidden())
```

Comment:
This will make the tagging not work when the bank tab tags are disabled. Maybe just check for the widget directly here instead of using tabInterface isHidden method

Reply:
true